contexts/security-v3-unstable.jsonld

```diff
@@ -84,87 +84,93 @@
         "@context": {
-            "@version": 1.1,
-            "@protected": true,
-            "id": "@id",
-            "type": "@type",
-            "challenge": "https://w3id.org/security#challenge",
-            "created": {
-              "@id": "http://purl.org/dc/terms/created",
-              "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
-            },
-            "domain": "https://w3id.org/security#domain",
-            "nonce": "https://w3id.org/security#nonce",
-            "proofPurpose": {
-              "@id": "https://w3id.org/security#proofPurpose",
-              "@type": "@vocab",
-              "@context": {
-                "@version": 1.1,
-                "@protected": true,
-                "id": "@id",
-                "type": "@type",
-                "sec": "https://w3id.org/security#",
-                "assertionMethod": {
-                  "@id": "https://w3id.org/security#assertionMethod",
-                  "@type": "@id",
-                  "@container": "@set"
-                },
-                "authentication": {
-                  "@id": "https://w3id.org/security#authenticationMethod",
-                  "@type": "@id",
-                  "@container": "@set"
-                }
+          "@version": 1.1,
+          "@protected": true,
+          "id": "@id",
+          "type": "@type",
+          "challenge": "https://w3id.org/security#challenge",
+          "created": {
+            "@id": "http://purl.org/dc/terms/created",
+            "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+          },
+          "domain": "https://w3id.org/security#domain",
+          "nonce": "https://w3id.org/security#nonce",
+          "proofPurpose": {
+            "@id": "https://w3id.org/security#proofPurpose",
+            "@type": "@vocab",
+            "@context": {
+              "@version": 1.1,
+              "@protected": true,
+              "id": "@id",
+              "type": "@type",
+              "sec": "https://w3id.org/security#",
+              "assertionMethod": {
+                "@id": "https://w3id.org/security#assertionMethod",
+                "@type": "@id",
+                "@container": "@set"
+              },
+              "authentication": {
+                "@id": "https://w3id.org/security#authenticationMethod",
+                "@type": "@id",
+                "@container": "@set"
               }
-            },
-            "proofValue": "https://w3id.org/security#proofValue",
-            "verificationMethod": {
-              "@id": "https://w3id.org/security#verificationMethod",
-              "@type": "@id"
             }
+          },
+          "proofValue": "https://w3id.org/security#proofValue",
+          "verificationMethod": {
+            "@id": "https://w3id.org/security#verificationMethod",
+            "@type": "@id"
+          }
         }
+      },
+      "x509Certificate": {
+        "@id": "https://w3id.org/security#X509Certificate"
+      },
```

Comment:
Can simplify to this, right?

```suggestion
      "x509Certificate": "https://w3id.org/security#X509Certificate",
```

Reply:
Yeap was just following the convention further up in the file but agree we can shorthand it